oio/cli/directory/directory.py

```diff
@@ -166,10 +185,54 @@ def take_action(self, parsed_args):
         mapping = self.get_prefix_mapping(parsed_args)
         mapping.load(read_timeout=parsed_args.meta0_timeout)
         moved = mapping.rebalance()
-        mapping.apply(moved, read_timeout=parsed_args.meta0_timeout)
+        self._apply_mapping(mapping, moved=moved,
+                            read_timeout=parsed_args.meta0_timeout)
         self.log.info("Moved %s", moved)
 
 
+class DirectoryRestore(DirectoryCmd):
```

Comment:
This command needs to be added to the setup.cfg in order to become available.

Reply:
Oops!